app/_how-tos/store-the-gateway-database-credentials-with-aws-secrets-manager.md

```diff
@@ -0,0 +1,144 @@
+      include_content: cleanup/third-party/aws
+      icon_url: /assets/icons/aws.svg
+    - title: Clean up Konnect environment
+      include_content: cleanup/platform/konnect
+      icon_url: /assets/icons/gateway.svg
+    - title: Destroy the {{site.base_gateway}} container
+      include_content: cleanup/products/gateway
+      icon_url: /assets/icons/gateway.svg 
+---
+
+## 1. Create a Docker network
+```sh
+docker network create kong-net
+```
+The Docker network will be used for communication between {{site.base_gateway}} and the database.
+
+## 2. Run the database
+Create the `kong-database` container for the PostgreSQL database: 
+```sh
+docker run -d --name kong-database \
+ --network=kong-net \
+ -p 5432:5432 \
+ -e "POSTGRES_USER=admin" \
+ -e "POSTGRES_PASSWORD=password" \
+ postgres:9.6
+```
+The username and password specified in this command are the PostgreSQL master credentials.
+
+## 3. Create environment variables
+Define the username and password to use to connect {{site.base_gateway}} to the database and store them in environment variables.
+```sh
+export KONG_PG_USER=kong
+export KONG_PG_PASSWORD=KongFTW
+```
+
+## 4. Create a database user
+Create a user in the PostgreSQL container, using the credentials defined in the previous step:
+```sh
+docker exec -it kong-database psql -U admin -c \
+ "CREATE USER ${KONG_PG_USER} WITH PASSWORD '${KONG_PG_PASSWORD}'"
+```
+
+## 5. Create a database
+Create a database named `kong`, with the user you created as the owner:
+```sh
+docker exec -it kong-database psql -U admin -c "CREATE DATABASE kong OWNER ${KONG_PG_USER};"
+```
+
+## 6. Create a secret in AWS Secrets Manager
+Use AWS CLI to create a new secret named `kong-gateway-database` containing the username and password you defined:
+```sh
+aws secretsmanager create-secret --name kong-gateway-database \
+ --description "Kong GW Database credentials" \
+ --secret-string '{"pg_user":"'${KONG_PG_USER}'","pg_password":"'${KONG_PG_PASSWORD}'"}'
+```
+
+## 7. Run the Kong migrations
+```sh
+docker run --rm \
+ --network=kong-net \
+ -e "KONG_DATABASE=postgres" \
+ -e "KONG_PG_HOST=kong-database" \
+ -e KONG_PG_USER \
+ -e KONG_PG_PASSWORD \
+ kong/kong-gateway:latest kong migrations bootstrap
+```
+{:.note}
+> Note: `kong migrations` does not support secrets managements, so this step passes the database credentials with environment variables.
+
+## 8. Start {{site.base_gateway}}
+Create the {{site.base_gateway}} container with your AWS credentials and the vault references in the environment:
+```sh
+docker run -d --name kong-gateway \
+ --network=kong-net \
+ -e "KONG_DATABASE=postgres" \
+ -e "KONG_PG_HOST=kong-database" \
+ -e AWS_ACCESS_KEY_ID \
+ -e AWS_SECRET_ACCESS_KEY \
+ -e AWS_REGION \
+ -e AWS_SESSION_TOKEN \
+ -e "KONG_PG_USER={vault://aws/kong-gateway-database/pg_user}" \
+ -e "KONG_PG_PASSWORD={vault://aws/kong-gateway-database/pg_password}" \
+ -e KONG_LICENSE_DATA \
+ kong/kong-gateway:latest
+```
```

Comment:
Add a sentence that describes what a user would see if this works.

I got a string of numbers, so I'm assuming it worked, but I don't know what those numbers mean. I also see `kong-database` in docker, so I know it worked. Maybe they can run `docker container ls` and they'll see the database in the list?